cmd/agentflow/main.go

```diff
@@ -77,6 +80,23 @@ func run() error {
 	}
 
 	if err := reload(); err != nil {
+		var diags diag.Diagnostics
+		if errors.As(err, &diags) {
+			bb, _ := os.ReadFile(configFile)
+
+			p := diag.NewPrinter(diag.PrinterConfig{
+				Color:              !color.NoColor,
```

Comment:
This gets set by the library automatically based on the environment it's running at. Do we want to negate it, or just set it to always true?

Reply:
Right, negating it was the intent so you don't get the color codes for things like logs sent to Loki. I don't think it should be always true.